rasa/shared/nlu/constants.py

```diff
@@ -1,10 +1,12 @@
 TEXT = "text"
 INTENT = "intent"
 RESPONSE = "response"
+RESPONSE_SELECTOR = "response_selector"
 INTENT_RESPONSE_KEY = "intent_response_key"
 ACTION_TEXT = "action_text"
 ACTION_NAME = "action_name"
 INTENT_NAME_KEY = "name"
+FULL_RETRIEVAL_INTENT_NAME_KEY = "full_retrieval_intent_name"
```

Comment:
why does this end with `_key`?

Reply:
To empathize that's it a name of a key in a certain dictionary, and shouldn't be confused with any other constant.
Do you think it's too much details for a name of a constant?